script/Arcueid/Day 9/09_00_ARC09_1.txt

```diff
@@ -32,7 +32,7 @@ A hot summer day.
 -- Page 2, Offset 21298.
 -- 青い空と　大きな大きな入道雲。
 The blue sky.
- Large, large clouds towering above.
+ Large, towering cumulonimbus clouds.
```

Comment:
That's a shame, because I still think it sounds great. Partly because the number of beats in the sentence feels better, but also because I feel there's intentional symbolism here. They're thunderheads; evocative of summer, but also suggest an impending storm. He could've just said 雲 but he didn't.

Reply:
Well, "thunderhead" is a much more appealing option, then. The problem for me is that cumulonimbus is a _very_ scientific word, and it doesn't fit in with a scene like this. 入道雲, on the other hand, is an old Japanese word without that scientific overtone.